bitnami/wavefront-adapter-for-istio/templates/NOTES.txt

```diff
@@ -19,6 +39,7 @@ APP VERSION: {{ .Chart.AppVersion }}
     echo "The renconciler is available at http://127.0.0.1:{{ .Values.service.port }}"
     kubectl port-forward svc/{{ template "common.names.fullname" . }} {{ .Values.service.port }}:{{ .Values.service.port }} &
```

Comment:
Replace references to `.Values.service.port` with `coalesce .Values.service.ports.grpc service.port`